### ISSUE TITLE
HW due over weekends appears under Monday

### ISSUE THREAD

**Issue by amazansky** [MyHomeworkSpace/client#142]
On a Friday, homework due over the coming weekend displays under the Monday column in HW view (soon to be Dashboard). I don't know how we'd go about changing the UI to make it work though. Perhaps we could make a "Weekend" column that only shows if you have stuff over the weekend?

**Comment by thatoddmailbox** [comment]
Yeah, this is somewhat confusing, and it was mainly done because I personally don't assign myself homework on Saturday and Sunday. Which, to be clear, is not really a good reasoning. :smile:

The main question I guess would be that if we did add a weekend column, where would it go? Above Today? What if you also had stuff that's overdue? I guess we could also rename "Monday"?

(I do agree that having the stuff that's due _before_ Monday in a column with a big title saying "Monday" is...an interesting design choice)

**Comment by amazansky** [comment]
Part of me wants to put it above Monday because it's logical order (Today, Weekend, Monday, Soon, Long-Term). However I also kinda feel that Monday should have some sort of main focus (e.g. being at the top of a column) because it's the start of the week. Do you have any thoughts? @thatoddmailbox 

Also, this raises a separate issue, but seeing "Soon" after "Monday" is a bit weird—on a Friday, "Soon" feels like it should be the weekend. So maybe on Fridays only, Weekend and Monday should be in the same column, then "Soon" gets renamed to something like "Next Week". But this also feels like we're creating a whole separate UI language for Fridays which could be confusing.

**Comment by thatoddmailbox** [comment]
@amazansky Yeah, I wouldn't want to switch the names around too much since I think it would get confusing for users, especially if the order changes too...on the other hand, I don't think there's really a better way of doing that.

Could we just add a subtitle to "Monday"? Something that says "(and over the weekend)"? Maybe that's too weird though.

**Comment by willbarkoff** [comment]
@amazansky @thatoddmailbox How would you feel about something like this?
<img width="430" alt="image" src="https://user-images.githubusercontent.com/10969136/95003029-c9031880-05a8-11eb-93ac-e51851649aa9.png">
